KsApi/extensions/ApolloClient+RAC.swift

```diff
@@ -55,8 +55,8 @@ extension ApolloClient {
           }
           observer.send(value: data)
           observer.sendCompleted()
-        case let .failure(error):
-          observer.send(error: .couldNotDecodeJSON(error))
+        case .failure:
+          observer.send(error: .couldNotParseJSON)
```

Comment:
yea so I'm not sure why we wouldn't want to send the error through.

Reply:
Yea so I guess my only suggestion against using the `EmptyResponseEnvelope` in `cancelBacking` and in `changeCurrency` is that a failed mutation can still return from GQL as a successful parseable response, which means it will contain parent key but `nil` value . In the case of `cancelBacking`:
```
    let dict: [String: Any?] = [
      "cancelBacking": [
        "clientMutationId": nil
      ]
    ]
```
will still be successful even though clientMutationId is `nil`, but 
```
   let dict: [String: Any?] = [
      "cancelBacking": nil
    ]
```
will be a successful response but we can consider it a failure because top level dict didn't return anything.

Really what I'm trying to figure out here is if a mutation fails server side, we will still get a successful response. The json may still be parsable, but how do we know it really did succeed? The `clientMutationId`? and top level dict in the response?

Reply:
Synced on a call about this and determined we are going to continue to use the `EmptyResponseEnvelope`.